### ISSUE TITLE
Border radius

### ISSUE THREAD

**Issue by floringit** [entria/react-native-view-overflow#10]
Hi, really useful component. But it seems that border radius is not working on android for ViewOverflow components. Do you have any idea?

**Comment by sibelius** [comment]
Can u share a sample repository with your issue?

**Comment by floringit** [comment]
Hi, unfortunately no. But I made a [screenshot](https://content.screencast.com/users/FlorinTest/folders/Jing/media/cbd12445-4241-41f5-830f-7e38544dd79b/00000012.png).

The style for ViewOverflow component:

`flex: 1,
        display: 'flex',
        flexDirection: 'column',
        minHeight: 200,
        backgroundColor: '#fff',
        padding: 24,
        elevation: 3,
        shadowColor: 'rgba(0,0,0,0.07)',
        shadowOffset: {width: 0, height: 4},
        shadowRadius: 12,
        shadowOpacity: 1,
        borderRadius: 10`

**Comment by floringit** [comment]
Yes. The grey box is a sample view woth border radius. The code is the same as yours above.

**Comment by floringit** [comment]
It actualy looks like this:

```
<ViewOferflow style={styles.ctn}>
    <View style={styles.thumbCtn}>
    </View>
</ViewOferflow>
```

```javascript
ctn: {
    flex: 1,
    display: 'flex',
    flexDirection: 'column',
    backgroundColor: '#fff',
    padding: 24,
    elevation: 2,
    shadowColor: 'rgba(0,0,0,0.07)',
    shadowOffset: {width: 0, height: 4},
    shadowRadius: 12,
    shadowOpacity: 1,
    borderRadius: 10
},
thumbCtn: {
    height: 200,
    elevation: 3,
    shadowColor: 'rgba(0,0,0,0.15)',
    shadowOffset: {width: 0, height: 6},
    shadowRadius: 14,
    shadowOpacity: 1,
    borderRadius: 10,
    backgroundColor: '#000',
    zIndex: 1,
    marginTop: -56,
    marginBottom: 24
}
```

**Comment by Summys** [comment]
I'm facing the same problem, did you find any solution ?

**Comment by floringit** [comment]
Hi. Yes, as workaround I added another child View inside ViewOverflow with full width an height and added border radius to that View.

**Comment by jgcmarins** [comment]
can we close this one?

**Comment by AlohaDM** [comment]
Hi, the borderRadius is working but not borderTopLeftRadius/borderTopRightRadius etc...
